seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java

```diff
@@ -426,7 +426,11 @@ private boolean isIssueWeAlreadyKnow(String threadName) {
                 || threadName.startsWith("MaintenanceTimer")
                 || threadName.startsWith("cluster-")
                 // Iceberg
-                || threadName.startsWith("iceberg");
+                || threadName.startsWith("iceberg")
+                // Hbase
+                || threadName.contains("htable")
+                || threadName.contains("Rpc")
+                || threadName.contains("RPC");
```

Comment:
`RPC` looks like a little common, could you share the detail of thread name?

Reply:
Thanks for your suggestions and I will continue to make revisions.